libs/shared/src/types/message-template/index.ts

```diff
@@ -8,3 +10,19 @@ export enum TextAlignEnum {
   LEFT = 'left',
   RIGHT = 'right',
 }
+
+export interface IEmailBlock {
+  type: EmailBlockTypeEnum;
+  content: string;
+  url?: string;
+  styles?: {
+    textAlign?: TextAlignEnum;
+  };
+}
+
+export interface ITemplateVariable {
+  type: TemplateVariableTypeEnum;
+  name: string;
+  required: boolean;
+  defaultValue?: string | boolean;
+}
```

Comment:
This is where I found more appropriate for the interfaces to live so their importing is more tidied.